reo/src/ghp.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import pandas as pd
+import copy
+from reo.utilities import MMBTU_TO_KWH, TONHOUR_TO_KWHT
+from reo.src.incentives import Incentives, IncentivesNoProdBased
+from reo.src.data_manager import big_number
+
+
+class GHPGHXInputs:
+    """
+    Map the GHPGHX inputs to the objects in this class
+    """
+
+    ground_k_by_climate_zone = {
+                                "1A": 1.029,
+                                "2A": 1.348,
+                                "2B": 0.917,
+                                "3A": 1.243,
+                                "3B": 1.364,
+                                "3C": 1.117,
+                                "4A": 1.023,
+                                "4B": 0.972,
+                                "4C": 1.418,
+                                "5A": 1.726,
+                                "5B": 1.177,
+                                "6A": 0.977,
+                                "6B": 0.981,
+                                "7A": 1.034,
+                                "7B": 1.508
+                                }
+
+    def __init__(self, inputs_dict, **kwargs):
+        #TODO Add ALL the input fields for the GHPGHX model
+        #   (remove all intermediate calculated fields based on inputs from here, so it's cleaner as the GHP POST)
+        self.heating_thermal_mmbtu_per_hr = inputs_dict["heating_thermal_load_mmbtu_per_hr"]
+        self.heating_thermal_kw = list(np.array(self.heating_thermal_mmbtu_per_hr) * MMBTU_TO_KWH)
+        self.cooling_thermal_ton = inputs_dict["cooling_thermal_load_ton"]
+        self.cooling_thermal_kw = list(np.array(self.cooling_thermal_ton) * TONHOUR_TO_KWHT)
+
+class GHPGHXOutputs:
```

Comment:
Since the class only has an init method it is only a data filter: it would be much more efficient to replace it with a function. However, is it even necessary (as a function)? Why not just use `outputs_dict` where ever GHPGHXOutputs is used?